### PR TITLE
qr code functionality working

### PR DESCRIPTION
**Pull request by Ewan1245**

### CODE REVIEW

File: site/src/components/GenQR.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    const generateQRUrl = () => {
+        //encode the url string
+        console.log(ingredients);
+
+        const url_ext = `${ingredients}/${cuisine == "" ? "undefined" : cuisine}`;
+        
+        const url = base_url + url_ext;
+
+        console.log(encodeURI(url));
```
Comment (OButterfield) [inline site/src/components/GenQR.jsx:16]: Does this still need to be here? or was it just for testing?

File: site/src/App.jsx
Comment (OButterfield) [inline site/src/App.jsx:1]: Are the comments just a temporary change until the chatbot is fully working or can they be deleted?

File: site/src/components/GenQR.jsx (hunk 1 above)
Comment (Ewan1245) [reply site/src/components/GenQR.jsx:16]: It prints out the URL that the qr code will direct you to so makes it easier to test that everything's working

File: site/src/App.jsx
Comment (Ewan1245) [reply site/src/App.jsx:1]: Comments were just temporary to get rid of some errors that popped up when trying to run the chatbot

Comment (OButterfield) [reply site/src/App.jsx:1]: okay great.

**Review by OButterfield — APPROVED** [review]

**Pull request merged**